lib/html-proofer/check.rb

```diff
@@ -22,7 +23,9 @@ def run
     end
 
     def add_issue(desc, line: nil, status: -1, content: nil)
-      @issues << Issue.new(@path, desc, line: line, status: status, content: content)
+      @issues << Issue.new(@path, desc, "#{self::class } failed", line: line, status: status, content: content)
+      @stat.findings.push(Issue.new(@path, desc, "#{self::class } failed", line: line, status: status, content: content))
+      @stat.print_finding if @options[:stat]
```

Comment:
We print new statement immediately only if stat option is checked, it is requirement of stat:

> INCREMENTAL OUTPUT: Each finding SHOULD be delimited by a new line. A computer program with STAT-compliant output SHOULD immediately output each finding as available (e.g. use flush()). This allows the consumer to begin processing the results right away.
https://github.com/fulldecent/structured-acceptance-test/blob/master/Stat-Output.md